src/components/SearchForm.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { TfiSearch } from "react-icons/tfi";
+import "../styles/SearchForm.css";
+
+function SearchForm({ searchText, setSearchText, onSubmit }) {
+  const handleInputChange = (event) => setSearchText(event.target.value);
+
+  return (
+    <div className="search-form" role="search">
+      <input
```

Comment:
don't forget that you can use a placeholder here. That way you won't have to worry about using a default value in your search state. e.g.
`placeholder="enter search term"`